relay-event-normalization/src/event.rs

```diff
@@ -2087,6 +2095,111 @@ mod tests {
         "###);
     }
 
+    #[test]
+    fn test_ai_metrics() {
```

Comment:
This test does not cover extracted metrics, right?

```suggestion
    fn test_ai_measurements() {
```